src/main/java/com/google/javascript/gents/CommentLinkingPass.java

```diff
@@ -102,54 +104,48 @@ private boolean hasRemainingComments() {
       return lastCommentIndex < comments.size() - 1;
     }
 
+    private Comment getCurrentComment() {
+      return comments.get(lastCommentIndex);
+    }
+
+    private Comment getNextComment() {
+      return comments.get(lastCommentIndex + 1);
+    }
+
     /** Returns the ending line number of the current comment */
     private int getLastLineOfCurrentComment() {
-      return comments.get(lastCommentIndex).location.end.line + 1;
+      return getCurrentComment().location.end.line + 1;
     }
 
     /** Returns the starting line number of the next comment */
     private int getFirstLineOfNextComment() {
-      return comments.get(lastCommentIndex + 1).location.start.line + 1;
+      return getNextComment().location.start.line + 1;
     }
 
     /** Shifts a new comment into the group */
-    private void forceAddCommentToGroup() {
+    private void addNextCommentTo(List<Comment> commentGroup) {
       if (hasRemainingComments()) {
-        group.add(comments.get(lastCommentIndex + 1));
+        commentGroup.add(getNextComment());
       }
       lastCommentIndex++;
     }
 
-    /**
-     * Shifts a new comment into the group.
-     * If the new comment is separated from the current one by at least a line, outputs the
-     * current group of comments.
-     */
-    private void addCommentToGroup(Node n) {
```

Comment:
I had a hard time understanding what this method was trying to do (the javadoc didn't quite match the name, and it's not obvious from the code that it "adds a comment to the group".   I ultimately decided since it was so hard to come up with an accurate name, that maybe this isn't the right type of abstraction (even though it is used twice).